src/app/content/components/Toolbar/StudyGuidesButton.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useSelector } from 'react-redux';
+import styled, { css } from 'styled-components/macro';
+import studyGuidesIcon from '../../../../assets/studyGuidesIcon.svg';
+import theme from '../../../theme';
+import { enableStudyGuides } from '../../selectors';
+import { toolbarIconStyles } from './iconStyles';
+import { PlainButton, toolbarDefaultText } from './styled';
+
+// tslint:disable-next-line:variable-name
+export const StudyGuidesWrapper = styled(PlainButton)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: 2rem;
```

Comment:
i've added a general `css` for toolbar buttons in my PR, since mine is simpler we could probably get that one merged first so you can use that css here

Reply:
I agree I was thinking about creating something like you did but didn't want to make more changes to keep it small. I think we should merge your pr first then :)